game/src/Combat/Stats.gd

```diff
@@ -51,3 +56,12 @@ func add_modifier(id: int, modifier) -> void:
 
 func remove_modifier(id: int) -> void:
 	modifiers.erase(id)
+
+
+func set_invulnerable_for_seconds(time: float) -> void:
```

Comment:
Nitpick but how about `set_invulnerable_for(duration: float) -> void:` or `duration_seconds` for the param name?

Reply:
The rationale is most properties called duration or something in Godot are in seconds. Like when you call `create_timer(duration)`